codec-redis/src/main/java/io/netty/handler/codec/redis/AbstractCollectionRedisMessage.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.AbstractReferenceCounted;
+import io.netty.util.ReferenceCountUtil;
+import io.netty.util.internal.ObjectUtil;
+import io.netty.util.internal.StringUtil;
+import io.netty.util.internal.UnstableApi;
+
+import java.util.Collection;
+
+/**
+ * Abstract class for Aggregate data types message.
+ */
+@UnstableApi
+public abstract class AbstractCollectionRedisMessage extends AbstractReferenceCounted
+    implements AggregatedRedisMessage {
+
+    protected Collection<RedisMessage> children;
```

Comment:
private final ?

Reply:
The sub class `ArrayRedisMessage` and `SetRedisMessage` need access this `children` field on the abstract method `children()`  for cast to `List<Message>` or `Set<Message>`, so it can't be private? but I had made it `final`